server/segment_rpc_test.go

```diff
@@ -386,31 +411,35 @@ func TestMakeFfmpegVideoProfiles(t *testing.T) {
 			Height:  int32(456),
 			Bitrate: int32(789),
 			Fps:     uint32(912),
+			FpsDen:  uint32(15),
 		},
 		{
 			Name:    "prof2",
 			Width:   int32(987),
 			Height:  int32(654),
 			Bitrate: int32(321),
 			Fps:     uint32(198),
+			//FpsDen:  0,
```

Comment:
Stray? It's ok to remove this comment and keep the zero value implicit

Reply:
I wanted to keep it explicit for the user but I see this might be confusing a few months down the line, will remove it and update the commit after testing manually :)